components/launcher/src/sys/unix/service.rs

```diff
@@ -115,6 +115,7 @@ impl Process {
 }
 
 pub fn run(msg: protocol::Spawn) -> Result<Service> {
+    debug!("spawning {}", msg.get_binary());
```

Comment:
Might be nice to have a corresponding line over in the Windows implementation, too.